addon/-private/execution_context/helpers.js

```diff
@@ -0,0 +1,37 @@
+import {
+  throwBetterError
+} from '../better-errors';
+
+/**
+ * @private
+ *
+ * Fills inputs, textareas, or contenteditable elements with the passed-in content.
+ *
+ * @param {jQuery} $selection              jQuery object containing collection of
+ * DOM elements to fill in
+ * @param {string} content                 Content to be inserted into target element(s)
+ * @param {Object} options                 Options for error reporting
+ * @param {string} options.selector        jQuery selector
+ * @param {Ceibo} options.pageObjectNode   PageObject node containing the method
+ * which, when invoked, resulted in this call to `fillElement`
+ * @param {string} options.pageObjectKey   Key of method on PageObject
+ * which, when invoked, resulted in this call to `fillElement`
```

Comment:
What's the proper indentation convention for wrapping lines in a comment like this?